src/reactviews/pages/QueryResult/table/plugins/headerFilter.plugin.ts

```diff
@@ -93,14 +103,22 @@ export class HeaderFilter<T extends Slick.SlickData> {
         )
             .addClass("slick-header-menubutton")
             .data("column", column);
+        if (
+            this.theme === ColorThemeKind.Dark ||
+            this.theme === ColorThemeKind.HighContrast
+        ) {
+            $el.addClass("vs-dark");
+        } else {
+            $el.addClass("vs-light");
+        }
```

Comment:
Do you know how `"vs-light"` differs from just `"light"`?  Could this just use `utils.resolveVscodeThemeType(themeKind)` instead?

Reply:
Quick question, why are we adding classes to the component?

Is using vscode css variables not enough in this case? We should generally just stick to them for all of our coloring and styling needs.

Currently, this doesn't support vs Hight Contrast light theme.

Reply:
This is because we in table.css we show either a light or dark version of the filter icon and this is controlled by these classes. I add the class like this now - any issues with this approach?
<img width="489" alt="Screenshot 2024-11-19 at 8 35 17 AM" src="https://github.com/user-attachments/assets/66a8fc8b-a16c-493d-a317-507ac9555e31">